backend/os/fileSystem.go

```diff
@@ -13,7 +14,10 @@ const Scheme = "file"
 const name = "os"
 
 // FileSystem implements vfs.Filesystem for the OS file system.
-type FileSystem struct{}
+type FileSystem struct {
+	client  s3iface.S3API
```

Comment:
This was probably a copy/paste mistake. I'm guessing the OS implementation doesn't need the S3 interface.

Reply:
Absolutely overzealous copy and paste. Before I go and remove the change entirely (tests still pass) is there any reason to keep the vfs.Options portion?